unittest/core_test.cpp

```diff
@@ -118,12 +146,14 @@ TEST_CASE(CoreFixture, R2IQTest)
         count = 0;
         totalsize = 0;
         radio->Start(decimate); // full bandwidth
-        std::this_thread::sleep_for(0.5s);
+        std::this_thread::sleep_for(1s);
         radio->Stop();
 
         REQUIRE_TRUE(count > 0);
         REQUIRE_TRUE(totalsize > 0);
         REQUIRE_TRUE(totalsize / count == transferSamples / 2);
+    printf("decimate=%d nxfers=%ld mindelay=%ld count=%u totalsize=%" PRIu64 "\n",
```

Comment:
space

Reply:
Gah, VS code not doing quite the right thing for me, will fix.